src/main/java/com/accelad/acctive/sim/kernel/core/evaluator/diff/FunctionKind.java

```diff
@@ -6,8 +6,8 @@
 
 enum FunctionKind {
     GREATER_THAN(new Function("greaterthan", 2)),
-    LOWER_THAN(new Function("lowerthan", 2)),
-    BETWEEN(new Function("between", 3)),
+    STRICTLY_LESS_THAN(new Function("lowerthan", 2)),
```

Comment:
I understand that right now, we won't change function text label to match the right name.
But we'll have to make that move at some point. Maybe open an issue to trace it?